packages/nextjs/utils/scaffold-eth/networks.ts

```diff
@@ -31,6 +31,8 @@ export const RPC_CHAIN_NAMES: Record<number, string> = {
   [chains.base.id]: "base-mainnet",
   [chains.baseGoerli.id]: "base-goerli",
   [chains.baseSepolia.id]: "base-sepolia",
+  [chains.celo.id]: "celo-mainnet",
+  [chains.celoAlfajores.id]: "celo-alfajores",
 };
```

Comment:
So this mapping is only used for creating alchemy url's, I think Alchemy doesn't support celo right? I couldn't find the celo rpc url on my alchemy dashboard

Reply:
no. Unfortunately, it does not support Celo yet. Is that needed to be added?